.github/workflows/managed.yml

```diff
@@ -23,6 +23,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        use-cache: [false]
```

Comment:
by default don't use cache, as of know it is faster and CI doesn't fail due to low disk space

Reply:
I'm not sure the cache is the problem. Issue is We mount the cache as a volume, and the copy it to and fro with docker cp thus doubling the size. In this [PR](https://github.com/percona/pmm/pull/1049) we removed the copy and it seems to be fine.

Reply:
It is a problem
![image](https://user-images.githubusercontent.com/91831719/180160948-ef5f9c36-d17a-448a-aa02-2124f109e8e4.png)

<img width="1247" alt="image" src="https://user-images.githubusercontent.com/91831719/180160828-fb103f5a-a970-45fa-97f5-9978fd720ba2.png">

Reply:
We at least need to run on machine with more disk space. Plus caching should be revisited, because it's slows things down

Reply:
Yeah, the screenshots were from an older run, but we can revisit caching later, I guess.

Reply:
Please create ticket for that

Reply:
@artemgavrilov sure